src/config/testnet_postake.mlh

```diff
@@ -1,10 +1,11 @@
+[%%import "config/coinbase/standard.mlh"]
+[%%import "config/consensus/postake_short.mlh"]
+[%%import "config/debug_level/logs.mlh"]
+
 [%%define genesis_ledger "testnet_postake"]
+[%%define genesis_state_timestamp "2019-01-30 12:00:00-08:00"]
+[%%define block_window_duration 20000]
```

Comment:
I think we want 60000 with snarks on for now

Reply:
Yes, good call.